src/infra/src/table/folders.rs

```diff
@@ -77,31 +88,33 @@ pub async fn list_dashboard_folders(org_id: &str) -> Result<Vec<Folder>, errors:
 
 /// Creates a new folder or updates an existing folder in the database. Returns
 /// the new or updated folder.
-pub async fn put(org_id: &str, folder: Folder) -> Result<Folder, errors::Error> {
+pub async fn put(
+    org_id: &str,
+    folder: Folder,
+    folder_type: FolderType,
+) -> Result<Folder, errors::Error> {
     let client = ORM_CLIENT.get_or_init(connect_to_orm).await;
 
-    let mut active: ActiveModel = match get_model(client, org_id, &folder.folder_id).await? {
```

Comment:
The logic in this block didn't change. It was just indented by the formatter after a new `folder_type` parameter was added to the `get_model` function call.